tests/cross/src/test/scala/tests/pc/CompletionDocSuite.scala

```diff
@@ -6,6 +6,11 @@ class CompletionDocSuite extends BaseCompletionSuite {
   override def requiresJdkSources: Boolean = true
   override def requiresScalaLibrarySources: Boolean = true
 
+  // the docs can change and fail the tests easily, Scala 3 changes much less often
```

Comment:
That's a tad ironic: https://github.com/scalameta/metals/actions/runs/7141590712/job/19449151785?pr=5927#step:4:2623 😂

Reply:
But that's because we keep breaking things around there :s